core/nwb.base.yaml

```diff
@@ -234,7 +234,10 @@ groups:
 - neurodata_type_def: Images
   neurodata_type_inc: NWBDataInterface
   default_name: Images
-  doc: A collection of images.
+  doc: An collection of images with an optional way to specify the order of the images.
+    For ordered image collections, if the images have different dimensions
+    and number of channels, use this type. If the images have the same dimensions and
+    number of channels, use the ImageStack type.
```

Comment:
```suggestion
    and number of channels, use this type. If the images are a time series of images with 
    the same dimensions and number of channels, use the ImageSeries type.
```

Reply:
Or should the last sentence refer to ``IndexSeries``? I just made the suggestion because I noticed that it still mentioned ``ImageStack``

Reply:
Good catch. I missed that reference

Reply:
Fixed